includes/class-newspack-popups-model.php

```diff
@@ -249,32 +250,16 @@ public static function retrieve_group_popups( $group_slugs, $include_unpublished
 		return self::retrieve_popups_with_query( new WP_Query( $args ) );
 	}
 
-	/**
-	 * Retrieve popups by IDs.
-	 *
-	 * @param  array   $ids array Array of popup IDs.
-	 * @param  boolean $include_unpublished Whether to include unpublished posts.
-	 * @return array Array of popup objects.
-	 */
-	public static function retrieve_popups_by_ids( $ids, $include_unpublished = false ) {
-		$args = [
-			'post_type'   => Newspack_Popups::NEWSPACK_POPUPS_CPT,
-			'post_status' => $include_unpublished ? [ 'draft', 'pending', 'future', 'publish' ] : 'publish',
-			'post__in'    => $ids,
-		];
-
-		return self::retrieve_popups_with_query( new WP_Query( $args ) );
-	}
-
 	/**
 	 * Get overlay test popups.
 	 *
+	 * @param  boolean $include_unpublished Whether to include unpublished posts.
 	 * @return array Overlay test popup objects.
 	 */
-	public static function retrieve_overlay_test_popups() {
+	public static function retrieve_overlay_test_popups( $include_unpublished = false ) {
```

Comment:
This method is no longer used with the argument

Reply:
Removed in a603104.